big_tests/tests/carboncopy_SUITE.erl

```diff
@@ -235,26 +240,19 @@ all_bobs_other_resources_get_sent_carbons([Alice, Bob1 | Bobs], Msg) ->
                         escalus_assert:has_no_stanzas(BobsResource) end,
     lists:foreach(GotCarbon, Bobs).
 
-carbons_get_disabled(ClientOrClients) ->
-    disable_carbons(ClientOrClients).
-
-carbons_get_enabled(ClientOrClients) ->
-    enable_carbons(ClientOrClients).
-
-
 %%
 %% Internal helpers
 %%
 
 %% Wrapper around escalus:story. Returns PropEr result.
 true_story(Config, UserSpecs, TestFun) ->
-    try   escalus_fresh:story_with_client_list(Config, UserSpecs, TestFun), true
+    try escalus_fresh:story_with_client_list(Config, UserSpecs, TestFun), true
     catch E ->
               {error, E}
     end.
 
 %% Number of resources per users
-no_of_resources() -> rand:uniform(4).
+no_of_resources() -> 1 + rand:uniform(4).
```

Comment:
Just a note: we could set it to deterministic values to avoid randomly passing/failing tests in case of a bug, e.g. `list:seq(2,5)`

Reply:
The idea is that tests should pass regardless of the number of resources, that's what proper is doing 🤔

Reply:
Ok let's keep it as it is.